frontend/src/hooks/useKeycloakWrapper.tsx

```diff
@@ -52,7 +52,7 @@ function useKeycloakWrapper(): IKeycloak {
    */
   const hasClaim = (claim?: string | Array<string>): boolean => {
     return (
-      !claim ||
+      claim !== undefined &&
```

Comment:
I like this change but please also check for `null`. Since you are using the strict equality here `!==` we are not checking for nulls. 

As is, this code will error out if null is passed as `null !== undefined && null.some()` will throw

Reply:
Good catch!  I’ll update